gclient/src/api/calls.rs

```diff
@@ -266,7 +266,7 @@ impl GearApi {
     pub async fn send_message_bytes_batch(
         &self,
         args: impl IntoIterator<Item = (ProgramId, impl AsRef<[u8]>, u64, u128)>,
-    ) -> Result<(Vec<Result<MessageId>>, H256)> {
+    ) -> Result<(Vec<Result<(MessageId, ProgramId)>>, H256)> {
```

Comment:
Why do we need this thing?
Any data we own before fn call shouldn't be returned as the result for me